src/components/ResourceCard/ResourceCard.styles.js

```diff
@@ -0,0 +1,58 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const Banner = styled.div`
+  opacity: 0.6;
```

Comment:
It is just working in mobile mode. Please add width = 100%; then it works in all viewport.